app/src/main/AndroidManifest.xml

```diff
@@ -18,7 +18,7 @@
 
         <meta-data
             android:name="com.google.android.geo.API_KEY"
-            android:value="${MAPS_API_KEY}" />
+            android:value="AIzaSyCvfF8uFvKe212J7GsKzhEMsw2x9-t3-PY" />
```

Comment:
Can you pls leave the variable as is and move the API key to ur local.properties. Not tryin to get charged by random people accessing this repo